composer.json

```diff
@@ -29,6 +29,19 @@
             "type": "composer",
             "url": "https://asset-packagist.org"
         },
+        {
+            "type": "package",
+            "package": {
+                "name": "library-moment/luxon",
+                "version": "1.11.2",
+                "type": "drupal-library",
+                "source": {
+                    "url": "https://github.com/bran/luxon",
```

Comment:
Why dependency from custom repo 
There is a main repo https://github.com/moment/luxon
IS there any PR you created with why your repo contains changes to main repo?
We'd like to push that PR to be approved into main repo instead of relying on custom fork. Maintenance wise we should add dependencies to mainstream repositories only.

Reply:
The reason I had to include a custom fork of Luxon is their repo doesn't hold a compiled browser friendly js source. My fork is simply theirs with the .js and .min.js built and committed. I had a lot of headaches trying to make composer post install/update run the npm build for it, that's why I included the fork instead.

The Luxon project also has a http-hosted, unversed "latest" .js source I can pull from instead.

@podarok what do you think the best approach is then in this case?